content_script.js

```diff
@@ -17,6 +17,27 @@ class HTMLSerializer {
     this.FILTERED_TAGS = new Set(['script', 'noscript', 'style', 'link']);
 
     /**
+     * @private {Set<string>} Contains the tag names for elements
+     *     that have no closing tags.
```

Comment:
I think I probably took it from the first list I found.   I've since changed it to reflect the spec.  Should I put the link you posted in a comment?